### ISSUE TITLE
Dependency problems

### ISSUE THREAD

**Issue by ingemar** [opal/opal-rails#35]
```
Bundler could not find compatible versions for gem "opal":
  In Gemfile:
    opal-rails (~> 0.7.0) ruby depends on
      opal-rspec (~> 0.4.0) ruby depends on
        opal (= 0.7.0) ruby

    opal (0.7.1)

**Comment by ingemar** [comment]
Why is `opal-rspec` even a dependency?

**Comment by adambeynon** [comment]
The version issue is my bad, `opal-rspec` and `opal-jquery` are both too strict on versioning, and can't work with opal `0.7.1`. Will push fixes asap.

**Comment by elia** [comment]
> Why is opal-rspec even a dependency?

For the same reason Rails ships with testing support out-of-the-box

https://github.com/opal/opal-rails#spec

😉

**Comment by ingemar** [comment]
Thanks @adambeynon

**Comment by elia** [comment]
Released opal-rspec 0.4.1
